internal/store/actions.go

```diff
@@ -161,3 +161,14 @@ type PanicAction struct {
 }
 
 func (PanicAction) Action() {}
+
+type TriggerTiltfileAction struct {
```

Comment:
ditto here, i'm not totally sure why we need a new action vs using the existing action, given how the codebase has been moving towards unifying these